### DISCUSSION TITLE
Flathub support

### DISCUSSION THREAD

**Discussion by dimii27** [neanes/neanes#816] [Ideas]
I am currently only learning about Linux so forgive me if I am being ignorant in any way.

Can you add neanes to the Flathub repository for ease of installation? Downloading the .appimage is easy, but one might forget to update the app and miss out on new features submitted to neanes

[Flathub documentation](https://docs.flathub.org/docs/for-app-authors/submission/)

**Comment by danielgarthur** [comment]
The appimage should automatically update already. See electron-builder docs [here](https://www.electron.build/auto-update).

If I recall correctly, you can run the appimage from the terminal and it will show you the logs for the autoupdater.

**Comment by danielgarthur** [comment]
I've confirmed that autoupdate is working for the AppImage (tested on Ubuntu 22). 

When you run the AppImage, it detects the newest available version, downloads it to a temporary location,  then when the AppImage is closed, the old version is deleted and the new version is copied to the same location from the temporary location.

I'll keep this discussion item around though, since Flatpak support is not out of the question, but I'm not sure when I would want to tackle this. Electron Builder supports exporting Flatpak files, but does not publish the files, so there is some work to be done there.

**Comment by ghost** [comment]
One of my main activities is pagination of Liturgical Services books, but without sheet music (at least for now).

Although I don't use this app at all, I have recommended it to all my friends from different countries.

Because I work exclusively on Linux, usually on different distributions, I found Flatpak to be the most convenient way to run applications distro agnostic.

A Father from the Monastery Vatoped from Mount Athos promised me that he would record and give me some high-quality voices to use in one of my projects, which I think can also be used in [neanes](https://github.com/neanes/neanes) instead of simple generated frequencies

For the next few years, I'm quite busy with several projects, but in the future, not in the near future, maybe I'll help with something in the  [neanes](https://github.com/neanes/neanes) project (at least with the Romanian translation of the application)

Instead of having different Organizations like [neanes](https://github.com/neanes) or [kolitsoy](https://github.com/kolitsoy), what do you think about an Organization on GitHub that would include all the projects related to the Orthodox Church (yours, mine and anyone else's)

**Reply by basil** [reply]
> Instead of having different Organizations like [neanes](https://github.com/neanes) or [kolitsoy](https://github.com/kolitsoy), what do you think about an Organization on GitHub that would include all the projects related to the Orthodox Church (yours, mine and anyone else's)

I am not sure this is such a good idea. I am experienced in open source project governance and community management, so I know that open source projects need [social infrastructure](https://producingoss.com/en/social-infrastructure.html) to survive and thrive. The governance model for the `neanes` organization is effectively one where Daniel acts as [BDFL](https://producingoss.com/en/benevolent-dictator.html) for the project. This is typical for early-stage projects and has served this project very well, but it does not scale to larger organizations with more people and subprojects. Typically the BDFL eventually moves on and grants decision-making authority to a group of successors. The success of this transition largely depends on the culture of the project and the ability of the BDFL/successors to define a governance model that is compatible with the project culture and can sustain the project going forward.

There are, of course, larger open source organizations that gather a wide variety of people and projects under one umbrella for legal and administrative purposes. Some examples that come to mind are the [Apache Software Foundation](https://www.apache.org/) and the [Eclipse Foundation](https://www.eclipse.org/). What you have in mind for Orthodox open source software sounds more like one of these foundations. Typically, these groups have much more formal process around membership and releases, as well as elections and project management committees to mediate disputes and enforce community guidelines.

Is such a foundation for Orthodox open source software achievable? Possibly, but IMHO it would fall far outside of the scope of this project. Such umbrella foundations require a significant amount of social and political infrastructure, and managing them is very time-consuming and difficult—perhaps even more difficult than writing a single package like Neanes. My recommendation, if you want to pursue this idea further, would be to study past precedent and start a new GitHub organization for the foundation you have in mind. Once you have a governance model in place, you could advertise the foundation to various existing projects to see if there is interest.

**Reply by ghost** [reply]
Thanks for the clarification, I have no experience in governing open source projects.

My main idea is to be an central source that contains information about all existing projects (for example, I found out about [neanes](https://github.com/neanes/neanes) about two years after its appearance only because I followed [@ilizol](https://github.com/ilizol)).

I think that even only creating a simple Repository that contains links and a little description of all these projects is enough.

**Reply by dimii27** [reply]
> I think that even only creating a simple Repository that contains links and a little description of all these projects is enough.

A massive list of free software and media is [FMHY](Https://fmhy.xyz) (and the [r/FMHY](https://reddit.com/r/fmhy) subReddit)

We could propose to add a section there, but it's pretty niche (in my opinion) to be included. Nonetheless, the repository you are suggesting could be added as an entry to the list. I'm sure it would bring a few new users if it would get that exposure.
